test/App.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestRenderer from 'react-test-renderer';
+import { mount, shallow } from 'enzyme';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import App from "../src/App";
+
+it('should render without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
+
+describe('App Snapshot', () => {
+  test('renders', () => {
```

Comment:
I'd suggest choosing one (`it` or `test`) and using everywhere.

Reply:
Changed.

Reply:
Removed duplicated test.